.travis.yml

```diff
@@ -21,13 +21,13 @@ script:
   - pyflakes dockerspawner
   - |
     # pre-pull images to avoid long wait times in tests
-    for v in 0.8 0.9; do
```

Comment:
Presumably the intention is to restore the for loop in future to check both 1.0 and 1.1 (as opposed to only 1.1)? If so I think keeping the for loop with just one item `for v in 1.0; do` will make it slightly easier for whoever fixes this in future.

Reply:
I just pushed the 1.1 image manually, so it should work now, and opened https://github.com/jupyterhub/jupyterhub/pull/3118 to fix it in the future

Reply:
Yaaay. Thank you :heart: 
I've just dropped the last commit that removed the tests for 1.1 and it seems that everything is green now.